### DISCUSSION TITLE
InitializeCompenent not auto-generated anymore?

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#9664] [Q&A] [answered]
I have tried the latest nightly build 11.0.999-cibuild0027335-beta and besides the missing IControl (vs. preview4) Visual Studio build sais that InitializeComponent is missing. So far I had not implemented that at it seems that it had always been auto-generated. Your sample apps in the Avalonia repo always implement InitializeComponent specifically calling the xaml load manually.

What is the strategy here?

**Comment by jp2masa** [comment]
I think I usually get that error when the XAML compilation fails, check the build log.

**Comment by maxkatz6** [comment, marked as answer]
You need to update XamlReferenceNameGenerator package to support a newer package.

**Reply by llfab** [reply]
```<PackageReference Include="XamlNameReferenceGenerator" Version="1.5.1" />``` does make it!
